.gitignore

```diff
@@ -31,7 +31,6 @@ custom-elements.json
 packages/*/src/**/*.css.js
 packages/*/custom-elements.json
 packages/**/*.js
-packages/**/spectrum-vars.json
```

Comment:
Legacy asset from the old vars and expressers packages; these are no longer generated in the workflow (even before this process-spectrum cutover).